go.mod

```diff
@@ -23,44 +21,44 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.0.1
 	github.com/kubernetes-csi/lib-volume-populator v1.2.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
-	github.com/openshift/api v0.0.0
-	github.com/openshift/client-go v0.0.0
+	github.com/onsi/gomega v1.24.1
+	github.com/openshift/api v0.0.0-20230406152840-ce21e3fe5da2
+	github.com/openshift/client-go v0.0.0-20230324103026-3f1513df25e0
 	github.com/openshift/custom-resource-status v1.1.2
-	github.com/openshift/library-go v0.0.0-20211220195323-eca2c467c492
+	github.com/openshift/library-go v0.0.0-20230328115725-6ed98e0ed0b9
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190725173916-b56e63a643cc
 	github.com/ovirt/go-ovirt v0.0.0-20210809163552-d4276e35d3db
 	github.com/ovirt/go-ovirt-client v0.9.0
 	github.com/ovirt/go-ovirt-client-log-klog v1.0.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.1
-	github.com/prometheus/client_model v0.2.0
+	github.com/prometheus/client_golang v1.14.0
+	github.com/prometheus/client_model v0.3.0
 	github.com/rs/cors v1.7.0
 	github.com/ulikunitz/xz v0.5.10
 	github.com/vmware/govmomi v0.23.1
-	go.uber.org/zap v1.19.1
-	golang.org/x/sys v0.5.0
+	go.uber.org/zap v1.24.0
+	golang.org/x/sys v0.6.0
 	google.golang.org/api v0.106.0
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/square/go-jose.v2 v2.5.1
-	k8s.io/api v0.25.0
-	k8s.io/apiextensions-apiserver v0.23.5
-	k8s.io/apimachinery v0.25.0
-	k8s.io/apiserver v0.23.0
+	k8s.io/api v0.26.3
+	k8s.io/apiextensions-apiserver v0.26.1
+	k8s.io/apimachinery v0.26.3
+	k8s.io/apiserver v0.26.1
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/cluster-bootstrap v0.0.0
-	k8s.io/code-generator v0.23.3
-	k8s.io/component-helpers v0.23.0
-	k8s.io/klog/v2 v2.70.1
-	k8s.io/kube-aggregator v0.23.0
-	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1
-	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
+	k8s.io/cluster-bootstrap v0.26.1
+	k8s.io/code-generator v0.26.1
+	k8s.io/component-helpers v0.26.1
```

Comment:
can swap the copied funcs like https://github.com/kubevirt/containerized-data-importer/blob/167a90ec528aef7150bbb54778b1c7362f7a0e37/pkg/controller/datavolume/util.go#L388

Reply:
we also copied CheckAccessModes :donthateme: